src/hub.js

```diff
@@ -307,6 +307,8 @@ async function runBotMode(scene, entryManager) {
 
 document.addEventListener("DOMContentLoaded", async () => {
   const scene = document.querySelector("a-scene");
+  scene.removeAttribute("keyboard-shortcuts"); // Remove F and ESC hotkeys from aframe
```

Comment:
That looks good. 

If we didn't remove the component like this, we might set the `keyboard-shortcuts` component data's `enterVR` property to `false` (which would be a worse solution). 
https://github.com/aframevr/aframe/blob/master/src/components/scene/keyboard-shortcuts.js#L16

Removing the component like you did here is better because there's no point running a tick that isn't doing anything.
We could do the same for `vr-mode-ui`, which is also added by `a-scene` (https://github.com/aframevr/aframe/blob/master/src/core/scene/a-scene.js#L60).

In hubs we disable this thru the `html`.

```
src/react-components/avatar-selector.js:154:        <a-scene vr-mode-ui="enabled: false" ref={sce => (this.scene = sce)}>
src/hub.html:32:        vr-mode-ui="enabled: false"
src/scene.html:18:        vr-mode-ui="enabled: false"
```

It would be better if we could tell aframe not to add this default component in the first place.

Reply:
I remember trying to do this and failing :(